pkg/test/constants.go

```diff
@@ -0,0 +1,6 @@
+package test
+
+const (
+	TestType    = "TEST_TYPE"
+	INTEGRATION = "INTEGRATION"
```

Comment:
```suggestion
	Integration = "INTEGRATION"
```

But maybe `E2e` would be better than Integration?